### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by fuomag9** [fuomag9/ChaturbateBot#15]
https://sentry.io/share/issue/a12ddcdb3b994baa82c6190981452736/
```
IndexError: list index out of range
  File "/root/ChaturbateBot/ChaturbateBot.py", line 113, in handle_add
    username=message.text.split(" ")[1]

IndexError: list index out of range
```

**Comment by liyiecho** [comment]
It should be check the length of `message.text.split(" ")` first.
```
    if len(message.text.split(" "))<2:
      risposta(message.chat.id, "You input was wrong.Please check again!")
      return
    username=message.text.split(" ")[1]
```

 The same in `handle_remove`.

I find a bug  when I remove anything like "1" not in my list, it always said "1 has been removed".

**Comment by liyiecho** [comment]
To use `for username in message.text.split(" ")[1:]:` instead of `username=message.text.split(" ")[1]` can be more quickly add many modles at once.

**Comment by fuomag9** [comment]
fixed in b6a46e861ea94473b74e896b0494bc90916d5ae0

**Comment by fuomag9** [comment]
also fixed in 5f2de23bf38a57596ef520468a2c92d1f8673e93
